app/controllers/api/v1/payments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::PaymentsController < ApplicationController
+	def order_create 
+		begin
+		    @user = User.find_by_id(params[:user_id])
+		    p @user
```

Comment:
Remove order creation code from payment controller, put this inside orders controller